cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1137,6 +1148,15 @@ private void deleteZnodesToStopExecution() {
             }
           }
         }
+        if (!slowTasksToReport.isEmpty()) {
```

Comment:
Nit: This blob seems to perform a specific task -- should we move it to a function?